### ISSUE TITLE
Godot crashes when attempting to create a window on a different monitor

### ISSUE THREAD

**Issue by MannyFluss** [godotengine/godot#93247]
### Tested versions

- reproduced in 4.2.2 stable release, 4.0.0 stable release.

### System information

Godot v4.2.stable - Windows 10.0.22631 - GLES3 (Compatibility) - NVIDIA GeForce RTX 3080 (NVIDIA; 31.0.15.4629) - Intel(R) Core(TM) i9-10900KF CPU @ 3.70GHz (20 Threads)

### Issue description

creating any popup on the secondary monitor crashes the godot editor.

### Steps to reproduce

create any popup menu on your second monitor. (add node, project settings... etc). 

https://github.com/godotengine/godot/assets/85276644/08c041cc-bd41-4e5c-b686-dd1e080ddc29

### Minimal reproduction project (MRP)

N/A

**Comment by matheusmdx** [comment]
I can't reproduce this issue on my pc

Godot v4.2.2.stable - Windows 10.0.19045 - GLES3 (Compatibility) - AMD Radeon RX 580 2048SP (Advanced Micro Devices, Inc.; 31.0.21912.14) - AMD Ryzen 5 3600 6-Core Processor (12 Threads)

**Comment by MannyFluss** [comment]
is there a way to run a more detailed diagnostic? I truly have no clue what causes this to happen but its really annoying.

**Comment by matheusmdx** [comment]
> is there a way to run a more detailed diagnostic? I truly have no clue what causes this to happen but its really annoying.

Yes, but that require the engine with debug symbols, so when the crash happens a backtrace will be generated and that can point the cause of the crash. You can compile by yourself following the steps described [here](https://docs.godotengine.org/en/stable/contributing/development/compiling/index.html) or if you don't feel confident to compile by yourself i can send you a compiled version with debug symbols

**Comment by MannyFluss** [comment]
yes that would be nice, ive tried to build it with debug symbols but I cant figure it out

**Comment by matheusmdx** [comment]
There is: https://drive.google.com/file/d/1tVu7G9ebf2wE7sgB5vHhju3_Lu1txMiP/view?usp=sharing

After you download that, open the directory where the exe is, hold shift and click with right mouse button in an empty space and select "open powershell console here". After that use `./godot.422.windows.editor.dev.x86_64.exe --debug`, with that when the crash happens a backtrace will be generated in the console
